package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "eagles-phantom-backend",
   "version": "1.0.0",
-  "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "babel-node server/app.js"
+    "dev": "cross-env NODE_ENV=development nodemon --exec babel-node ./server/app.js"
```

Comment:
why do you have two devs in your scripts?try to use one

Reply:
The first dev is no longer in the code, that - shows that it has been removed.